18-Nov-2022 00:57:27.521 GRAVE [http-nio-8080-exec-78] org.apache.catalina.core.StandardWrapperValve.invoke El Servlet.service() para el servlet [jsp] en el contexto con ruta [/PruebaJSP] lanzó la excepción [Ha sucedido una excepción al procesar la página JSP [/EditarPersona.jsp] en línea [26]

23:                 <form class="form-control" style="background-color:lavender" action="SvModificar"  method="POST">
24:                     <% Persona modificarPer = (Persona) request.getSession().getAttribute("perModificar");
25:                     %>
26:                     <p><label>ID: </label><input class="form-control" value="<%=modificarPer.getId()%>" type="text" name="idM"></p>
27:                     <p><label>DNI: </label><input class="form-control" value="<%=modificarPer.getDni()%>" type="text" name="dni"></p>
28:                     <p><label>Nombre: </label><input class="form-control" value="<%=modificarPer.getNombre()%>" type="text" name="nombre"></p>
29:                     <p><label>Apellido: </label><input class="form-control" value="<%=modificarPer.getApellido()%>" type="text" name="apellido"></p>


Stacktrace:] con causa raíz
	java.lang.NullPointerException
		at org.apache.jsp.EditarPersona_jsp._jspService(EditarPersona_jsp.java:139)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:765)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:465)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:383)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:331)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:765)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:543)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:698)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:367)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:639)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:885)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1693)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.lang.Thread.run(Thread.java:750)
18-Nov-2022 01:00:45.099 INFORMACIÓN [localhost-startStop-2] org.apache.catalina.core.ApplicationContext.log SessionListener: contextDestroyed()
18-Nov-2022 01:00:45.099 INFORMACIÓN [localhost-startStop-2] org.apache.catalina.core.ApplicationContext.log ContextListener: contextDestroyed()
